go/vt/discovery/healthcheck.go

```diff
@@ -140,7 +141,7 @@ const (
 // ParseTabletURLTemplateFromFlag loads or reloads the URL template.
 func ParseTabletURLTemplateFromFlag() {
 	tabletURLTemplate = template.New("")
-	_, err := tabletURLTemplate.Parse(TabletURLTemplateString)
+	_, err := tabletURLTemplate.ParseFromTrustedTemplate(uncheckedconversions.TrustedTemplateFromStringKnownToSatisfyTypeContract(TabletURLTemplateString))
```

Comment:
We use use the `uncheckedconversions` escape hatch here since we have this input from a command line setting. There's no way to avoid this, but to trust the input parameters.